### ISSUE TITLE
Bug: OBJ fails to load when the .mtl file contains "aniso" or "anisor"

### ISSUE THREAD

**Issue by TenHenEggs** [assimp/assimp#5635]
**Describe the bug**
Trying to import an OBJ file with its .mtl containing "aniso" or "anisor" fails with the message:
`Cannot parse string "niso" as a real number: does not start with digit or decimal point followed by digit.`

**To Reproduce**
Try to import [this](https://github.com/tinyobjloader/tinyobjloader/blob/release/models/pbr-mat-ext.mtl) model from [tinyobjloader](https://github.com/tinyobjloader/tinyobjloader)

**Expected behavior**
Model is successfully imported

**Platform**
 - OS: Windows
 - Version 5.4.0
 - Installed via vcpkg

**Comment by kimkulling** [comment]
That is right. I have just checked the implementation and the parser is just ignoring ansio.

**Comment by SharpieLaBeouf** [comment]
I am having the same issue. I have exported my wavefront model from Blender using the PBR Extensions export option for [wavefront materials](https://en.wikipedia.org/wiki/Wavefront_.obj_file#Physically-based_rendering). I have received this error when attempting to load using ASSIMP.

Assimp Scene Invalid: Cannot parse string "niso" as a real number: does not start with digit or decimal point followed by digit.

**Comment by GitHubDragonFly** [comment]
@TenHenEggs and @SharpieLaBeouf maybe also check the issue https://github.com/assimp/assimp/issues/5554

It might show you possible ways of fixing OBJ + MTL issues.

**Comment by tellypresence** [comment]
@TenHenEggs Is this still a problem?

**Comment by TenHenEggs** [comment]
@tellypresence 
On release 5.4.3 bug still happens.
I also managed to compile master ac59884, and it doesn't solve the issue either.

**Comment by turol** [comment]
Link points to .mtl and not the .obj. Can you link to both for ease of downloading?

**Comment by TenHenEggs** [comment]
Sure,
[pbr-mat-ext.mtl](https://github.com/tinyobjloader/tinyobjloader/blob/release/models/pbr-mat-ext.mtl)
[pbr-mat-ext.obj](https://github.com/tinyobjloader/tinyobjloader/blob/release/models/pbr-mat-ext.obj)

Zip file with both files:
[pbr-mat-ext.zip](https://github.com/user-attachments/files/19822308/pbr-mat-ext.zip)

**Comment by turol** [comment]
I think this is the relevant piece of code:
https://github.com/assimp/assimp/blob/ac5988422a7f1898b25c13fb5fb068c646a4c544/code/AssetLib/Obj/ObjFileMtlImporter.cpp#L266

Looks like it needs something to skip to the next space instead of just the next character.

**Comment by dimitre** [comment]
+1 for this one.

**Comment by tellypresence** [comment]
OBJ import issues tracked via epic #6179

**Comment by dimitre** [comment]
Assimp now errors when finding this data keys. it would be great if it could ignore, or output a warning instead

**Comment by paulbauriegel** [comment]
I actually had a similar issue or something related to this here: https://github.com/assimp/assimp/pull/6253 so I added a fix for aniso & anisor there too.

**Comment by kimkulling** [comment]
Closed by https://github.com/assimp/assimp/pull/6253

**Comment by kimkulling** [comment]
Closed by https://github.com/assimp/assimp/pull/6452
